docs/modules/migrate/pages/rolling-restart.adoc

```diff
@@ -0,0 +1,45 @@
+= Upgrade Cluster to {enterprise-product-name}
+:description: You can use an in-place rolling restart to move each member on your Open Source cluster to an Enterprise cluster.
+
+{description}
+
+This approach ensures that you can move your cluster without interruption to the service. 
+Similar to the rolling upgrade process, a rolling restart allows you to move one member at a time.
+
+The codebase of your {open-source-product-name} cluster must match the target {enterprise-product-name} release. 
+Currently, rolling restart can only be used to update your 5.5 {open-source-product-name} cluster members.
+
+== Best Practice
+
+Before starting a rolling restart, consider the following best practices:
+
+* Backup your IMap and ReplicatedMap data structures. This is not essential, but it is recommended practice
```

Comment:
How should they do the backup? AFAIK we don't provide any tooling for that. (DMT can transfer data between clusters, but not simply export...)

Reply:
I have added "using your usual method", but if this is too much of a fudge (or not something that is generally/easily done) I'd suggest removing this bullet from the list.

Reply:
I suppose that's good.